src/modules/commander/state_machine_helper.cpp

```diff
@@ -473,6 +474,8 @@ bool set_nav_state(vehicle_status_s *status, actuator_armed_s *armed, commander_
 
 	case commander_state_s::MAIN_STATE_POSCTL: {
 
+			const bool rc_fallback_allowed = (posctl_nav_loss_act != position_nav_loss_actions_t::LAND_TERMINATE) || !is_armed;
```

Comment:
May I ask what use case the `posctl_nav_loss_act` option is for? Independently of this pr I suggest removing it to reduce complexity because I don't see which use case is breaking or if so we should cover it differently by default if possible.